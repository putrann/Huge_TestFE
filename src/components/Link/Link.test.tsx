import * as React from 'react';
import { render } from '@testing-library/react';
import { Link } from '.';

describe('<Link/>', () => {
  it('Should match snapshot', () => {
    const { container } = render(<Link msg="Learn more" />);
    expect(container.firstChild).toMatchSnapshot();
  });

  it('Should not render anything if type mismatch or undefined', () => {
    // @ts-ignore
    const { container } = render(<Link />);
    expect(container.firstChild).toBeNull();
  });
});
