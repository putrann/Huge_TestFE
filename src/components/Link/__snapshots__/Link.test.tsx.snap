// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Link/> Should match snapshot 1`] = `
<a
  href="https://www.google.com"
  target="_blank"
>
  <div
    style="margin-right: 8px;"
  >
    Learn more
     
  </div>
  <svg
    fill="currentColor"
    height="1em"
    width="1em"
  >
    right.svg
  </svg>
</a>
`;
